doc/release/1.13.0-notes.rst

```diff
@@ -83,6 +83,9 @@ for instance). Note that this does not remove the need for Mingwpy; if you make
 extensive use of the runtime, you will most likely run into issues_. Instead,
 it should be regarded as a band-aid until Mingwpy is fully functional.
 
+The MinGW toolset can now also be used together with the moveable WinPython 3.4
+distribution, for programs that uses a PySide Qt4 front-end.
```

Comment:
Agreed. It was meant as a follow-up on the preceeding paragraph, since this changeset in a way is a follow-up to the pull request that added that one - but the reader of the change file doesn't know that!

I have added a more targeted description of the use case (building extensions).